frontend/src/features/conversation/ReplyBox.vue

```diff
@@ -151,6 +152,15 @@ const { uploadingFiles, handleFileUpload, handleFileDelete, mediaFiles, clearMed
     linkedModel: 'messages'
   })
 
+// Setup draft management composable
+const currentDraftKey = computed(() => conversationStore.current?.uuid || null)
+const {
+  htmlContent: localHtmlContent,
+  textContent: localTextContent,
```

Comment:
If you don't alias it.


Then most of the code remains unchanged. Previously var was named `htmlContent`